website/docs/cloud-docs/projects/manage.mdx

```diff
@@ -0,0 +1,120 @@
+---
+page_title: Manage projects in HCP Terraform and Terraform Enterprise
+description: |-
+  Use projects to organize and group workspaces and create ownership boundaries
+  across your infrastructure.
+---
+
+# Manage projects
+
+This topic describes how to create and manage projects in HCP Terraform and Terraform Enterprise. A project is a folder containing one or more workspaces.
+
+## Requirements
+
+You must have the following permissions to manage projects:
+
+- You must be a member of a team with the **Manage all Projects**  permissions enabled to create a project. Refer to [Organization Permissions](/terraform/cloud-docs/users-teams-organizations/permissions#organization-permissions) for additional information.
+- You must be a member of a team with the update team access to add a new team access for organization-visible teams for a project.
+- You must be a member of a team with update and delete permissions to be able to update and delete teams respectively.
```

Comment:
This permission doesn't seem to affect projects at all, just teams? Maybe we can remove it?

Reply:
It's possible that this has changed and the required permissions were never updated: https://developer.hashicorp.com/terraform/cloud-docs/projects/managing#permissions

@brandonc - is this still accurate?